geoips/interfaces/base.py

```diff
@@ -25,7 +25,7 @@
 LOG = logging.getLogger(__name__)
 
 plugin_deprecations = True
-if geoips_version >= "2.0.0":
+if geoips_version >= "1.10.0":
```

Comment:
Note I am forcing this to >= 1.10.0 for now for testing purposes - we can decide if we want to keep that or allow backwards compatibility until 2.0 when we are ready to merge.